docs/developer/price-expiration.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Price Expiration
+sidebar_label: Price Expiration
+---
+
+:::info
+This feature was introduced in **Saleor 3.21**. In previous versions, the base prices were refreshed with every single recalculation.
+:::
+
+## Introduction
+When adding a line to an order, the product variant price is taken from the relevant channel listing. 
+However, throughout the order lifecycle, price recalculations may be required, and channel listing prices can be updated in the meantime.
+To ensure price consistency over time, regardless of channel listing updates, Saleor provides a settings that allows to control the price freeze period.
```

Comment:
I have a trouble to understand what do you want to tell us.
`price recalculations may be required` - because of what?
`and channel listing prices can be updated in the meantime.` - how it's related with other part of the sentence?

> To ensure price consistency over time, regardless of channel listing updates, Saleor provides a settings that allows to control the price freeze period.

I don't think that mentioning here channel listings update is important. 
I think that introduction should say what happened in general, not going into the details

Reply:
> I don't think that mentioning here channel listings update is important.

This is the essence of this change - we don't fetch prices from channel listing, but we use denormalized ones.

> price recalculations may be required - because of what?

 As you said, this is introduction so it shouldn't got much into details what can trigger the price recalculation